### DISCUSSION TITLE
To launch Safari browser in iPhone using appium, should we need an apple account with Development Teams mapped to it?

### DISCUSSION THREAD

**Discussion by narasimhanukala** [appium/appium#14954] [Q&A]
To launch Safari browser in iPhone using appium, should we need an apple account with Development Teams mapped to it?

i have configured following capabilities in Appium Desktop(1.19.1), and start session
{
  "deviceName": "iPhone",
  "udid": "xxxxxxxxxxxxxxxxxxxxxx",
  "platformName": "iOS",
  "platformVersion": "14.3",
  "browserName": "safari",
  "autoLaunch": true,
  "useNewWDA": true,
  "startIWDP": true
}

noticed following exception:

WebDriverAgentRunner:
[MJSONWP] 		WebDriverAgentRunner-Runner.app encountered an error (Failed to install or launch the test runner. If you believe this error represents a bug, please attach the result bundle at /Users/yadu/Library/Developer/Xcode/DerivedData/WebDriverAgent-alwvnomvwrdtzoaxbbkniqrpcdpp/Logs/Test/Test-WebDriverAgentRunner-2020.12.16_23-38-25-+0800.xcresult. (Underlying Error: The file “WebDriverAgentRunner-Runner.app” couldn’t be opened because there is no such file. The file doesn’t exist. (Underlying Error: The operation couldn’t be completed. No such file or directory)))
[MJSONWP] * TEST EXECUTE FAILED *
[MJSONWP] Testing started. Make sure you follow the tutorial at https://github.com/appium/appium-xcuitest-driver/blob/master/docs/real-device-config.md. Try to remove the WebDriverAgentRunner application from the device if it is installed and reboot the device.
[MJSONWP]     at quitAndUninstall (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/lib/driver.js:541:15)

**Comment by KazuCocoa** [comment]
Yes, to run WDA on a real device. Previously the provisioning was enough with a personal account, but it seems they require a paid one. http://appium.io/docs/en/drivers/ios-xcuitest-real-devices/#troubleshooting-ideas

Or you can try out http://appium.io/docs/en/drivers/safari/index.html for Appium 1.20+.
It uses `safaridriver`, so such WDA does not need.
